src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewTemplate.cs

```diff
@@ -23,86 +22,25 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Options;
 using OrchardCore.DisplayManagement.Layout;
-using OrchardCore.DisplayManagement.Liquid.Filters;
 using OrchardCore.DisplayManagement.Liquid.Internal;
-using OrchardCore.DisplayManagement.Liquid.Tags;
 using OrchardCore.DisplayManagement.Shapes;
-using OrchardCore.DisplayManagement.Zones;
-using OrchardCore.DynamicCache.Liquid;
 using OrchardCore.Environment.Shell.Scope;
 using OrchardCore.Liquid;
 
 namespace OrchardCore.DisplayManagement.Liquid
 {
-    public class LiquidViewTemplate : BaseFluidTemplate<LiquidViewTemplate>
+    public class LiquidViewTemplate
     {
+        public static LiquidViewParser Parser = new LiquidViewParser();
```

Comment:
I'm looking for this `_liquidViewParser.RegisterTag` but this static was all I found ;)

Reply:
LiquidTemplate.Parser, yeah still static, need to remove LiquidTemplate altogether for this change. Will do with the other things I mentioned should be improved. But it can be done now, before it couldn't... at least